kyo-core/jvm/src/main/scala/kyo/Process.scala

```diff
@@ -0,0 +1,235 @@
+package kyo
```

Comment:
I tried to keep the file rename in the git history but I couldn't make it work :(

Reply:
It seems it actually worked but github doesn't show it: https://github.com/getkyo/kyo/pull/471/commits/d2b781bca6d72aca0d3276c8a0fd2b6c7b35007b